src/main/kotlin/lotto/LottoRank.kt

```diff
@@ -0,0 +1,18 @@
+package lotto
+
+enum class LottoRank(
+    val count: Int,
+    val prizeMoney: Int
+) {
+    THREE_COLLECT(3, 5_000),
+    FOUR_COLLECT(4, 50_000),
+    FIVE_COLLECT(5, 1_500_000),
+    SIX_COLLECT(6, 2_000_000_000),
+    NON_COLLECT(0, 0);
+
+    companion object {
+        fun getPrizeMoney(count: Int): Int {
```

Comment:
```suggestion
        fun getPrizeMoney(count: Int): LottoRank {
```
당첨금에 대한 정보를 LottoRank의 prizeMoney로 관리하고 계십니다.
그렇다면 LottoRank를 활용해보면 어떨까요?

Reply:
LottoRank 로 수정하였습니다~! 해당 과정에서 변경되는 코드 및 테스트 코드 수정하였습니다!